modules/consentManagement.js

```diff
@@ -42,27 +42,43 @@ const cmpCallMap = {
  */
 function lookupIabConsent(cmpSuccess, cmpError, adUnits) {
   let cmpCallbacks;
+  let cmpResponse = {};
+
+  // to collect the consent information from the user, we perform two calls to the CMP:
+  // first to collect the user's consent choices represented in an encoded string (via getConsentData)
+  // second to collect the user's full unparsed consent information (via getVendorConsents)
+  callIabCMP('getConsentData', function (consentResponse) {
+    cmpResponse.getConsentData = consentResponse;
+
+    callIabCMP('getVendorConsents', function (consentResponse) {
```

Comment:
why are you doing this serially and not in parallel ?

Reply:
I can chat about this with you if you're interested.

Reply:
yeah I think parallel would be ideal, especially if these are making ajax requests.  If they're not, it probably doesn't matter.

Reply:
I don't think these are making ajax requests - so it probably doesn't matter so much.